projects/igniteui-angular/src/lib/core/styles/components/calendar/_calendar-theme.scss

```diff
@@ -322,11 +322,11 @@
     }
 
```

Comment:
You missed the previous $border-radius on line 321

Reply:
It should be fixed now.